studies/views/submitted_studies_experiments.py

```diff
@@ -31,7 +31,9 @@ class SubmittedStudyExperiments(StudyRelatedPermissionsViewMixin, ModelViewSet,
     queryset = Experiment.objects.select_related("study", "study__approval_process", "study__submitter")
 
     def get_queryset(self):
-        qs = super().get_queryset().filter(study=self.kwargs.get("study_pk")).filter(study__submitter=self.request.user)
+        qs = super().get_queryset().filter(study=self.kwargs.get("study_pk"))
+        if not (hasattr(self.request.user, "profile") and self.request.user.profile.is_reviewer):
```

Comment:
ואם כן?

Reply:
then we don't need the extra filtering.. I've added a comment